packages/stark-ui/karma.conf.js

```diff
@@ -8,37 +8,57 @@ const defaultKarmaConfig = require("./node_modules/@nationalbankbelgium/stark-te
+	// adapt the resolution of the 3rd party modules used in stark-core
+	"@angularclass/hmr": "../stark-core/node_modules/@angularclass/hmr/dist/index.js",
+	"@ng-idle/core": "../stark-core/node_modules/@ng-idle/core/bundles/core.umd.js",
+	"@ng-idle/keepalive": "../stark-core/node_modules/@ng-idle/keepalive/bundles/keepalive.umd.js",
+	"@ngrx/store": "../stark-core/node_modules/@ngrx/store/bundles/store.umd.js",
+	"@ngrx/effects": "../stark-core/node_modules/@ngrx/effects/bundles/effects.umd.js",
+	"@ngx-translate/core": "../stark-core/node_modules/@ngx-translate/core/bundles/ngx-translate-core.umd.js",
+	"@uirouter/angular": "../stark-core/node_modules/@uirouter/angular/_bundles/ui-router-ng2.js",
+	"@uirouter/core": "../stark-core/node_modules/@uirouter/core/lib/index.js",
+	"@uirouter/rx": "../stark-core/node_modules/@uirouter/rx/lib/index.js",
+	cerialize: "../stark-core/node_modules/cerialize/index.js",
+	"class-validator": "../stark-core/node_modules/class-validator/index.js",
+	"deep-freeze-strict": "../stark-core/node_modules/deep-freeze-strict/index.js",
+	moment: "../stark-core/node_modules/moment/moment.js",
+	"lodash-es": "../stark-core/node_modules/lodash-es/lodash.js",
+	"lodash-es/cloneDeep": "../stark-core/node_modules/lodash-es/cloneDeep.js",
+	"lodash-es/findIndex": "../stark-core/node_modules/lodash-es/findIndex.js",
+	"lodash-es/floor": "../stark-core/node_modules/lodash-es/floor.js",
+	"lodash-es/get": "../stark-core/node_modules/lodash-es/get.js",
+	"lodash-es/isEmpty": "../stark-core/node_modules/lodash-es/isEmpty.js",
+	"lodash-es/isEqual": "../stark-core/node_modules/lodash-es/isEqual.js",
+	"lodash-es/merge": "../stark-core/node_modules/lodash-es/merge.js",
+	"lodash-es/noop": "../stark-core/node_modules/lodash-es/noop.js",
+	"lodash-es/sortBy": "../stark-core/node_modules/lodash-es/sortBy.js",
+	"lodash-es/startCase": "../stark-core/node_modules/lodash-es/startCase.js",
+	"lodash-es/uniqueId": "../stark-core/node_modules/lodash-es/uniqueId.js",
+	ibantools: "../stark-core/node_modules/ibantools/build/ibantools.js"
 };
 
 // start customizing the KarmaCI configuration from stark-testing
 const starkUiSpecificConfiguration = Object.assign({}, defaultKarmaConfig, {
 	// change the module resolution for the KarmaTypescript bundler
-	karmaTypescriptConfig: Object.assign(defaultKarmaConfig.karmaTypescriptConfig, {
-		bundlerOptions: Object.assign(defaultKarmaConfig.karmaTypescriptConfig.bundlerOptions, karmaTypescriptBundlerAliasResolution)
-	}),
+	karmaTypescriptConfig: {
+		...defaultKarmaConfig.karmaTypescriptConfig,
+		bundlerOptions: {
+			...defaultKarmaConfig.karmaTypescriptConfig.bundlerOptions,
+			resolve: {
+				alias: karmaTypescriptBundlerAlias
+			},
+			transforms: [
+				require("@nationalbankbelgium/stark-testing/node_modules/karma-typescript-angular2-transform"),
+				require("@nationalbankbelgium/stark-testing/node_modules/karma-typescript-es6-transform")({
```

Comment:
Can you add these to the exports of this file so that you can access it in `stark-ui/karma.conf.ci.js` instead of duplicating this block in these 2 files?

Reply:
Same as above. 😄

Reply:
Mmm well I don't think that this will change that much when #1145 is solved right? We would need to apply the same logic here... so we could re-use it here instead of duplicating it